components/widget/infobox/widget_infobox_center.lua

```diff
@@ -18,7 +18,6 @@ local Widget = Lua.import('Module:Widget')
 local Center = Class.new(
 	Widget,
 	function(self, input)
-		self.children = input.children or input.content or {}
```

Comment:
we still need `self.children = input.children`, right?
or does the Widget base class already do that?
(on phone atm, so shit too look up)

Reply:
Widget base does it